source/MonoGame.Extended.Particles/ParticleExtensions.cs

```diff
@@ -48,11 +48,11 @@ private static unsafe void UnsafeDraw(SpriteBatch spriteBatch, ParticleEmitter e
 
                 var position = new Vector2(particle->Position.X, particle->Position.Y);
                 var scale = particle->Scale;
-                var particleColor = new Color(color, particle->Opacity);
+                color.A = (byte)MathHelper.Clamp(particle->Opacity*255, 0, 255);
```

Comment:
The code here was refactored to remove the intermediate particleColor.
Color(Color, float) .ctor  is an MG extension. It doesn't exist in FNA.